axmol build error

- axmol version: axmol/dev  2.1.2
- devices test on:
- developing environments
   - NDK version: r23c
   - Xcode version: 14.2
   - Visual Studio: 
      - VS version:  2022 (17.6+)
      - MSVC version: 1929, 1934, 19.35, 19.36, 19.37
      - Windows SDK version: 10.0.22621.0
   - cmake version: 
Steps to Reproduce:

1、make a project
2、use AndroidStudio to open project and  compile project
In file included from E:/axmol/extensions/scripting/lua-bindings/manual/platform/android/jni/LuaBridge.cpp:32:
In file included from E:/axmol/extensions/scripting/lua-bindings/proj.android/../../../../core\base/UTF8.h:37:
In file included from E:/axmol/extensions/scripting/lua-bindings/proj.android/../../../../core\platform/android/jni/JniHelper.h:37:
In file included from E:/axmol/extensions/scripting/lua-bindings/proj.android/../../../../core\math/Vec3.h:496:
In file included from E:/axmol/extensions/scripting/lua-bindings/proj.android/../../../../core\math/Vec3.inl:22:
In file included from E:/axmol/extensions/scripting/lua-bindings/proj.android/../../../../core\math/Mat4.h:26:
In file included from E:/axmol/extensions/scripting/lua-bindings/proj.android/../../../../core\base/Macros.h:37:
E:/axmol/extensions/scripting/lua-bindings/proj.android/../../../../core\base/Console.h:46:10: fatal error: 'yasio/io_watcher.hpp' file not found
#include "yasio/io_watcher.hpp"
         ^~~~~~~~~~~~~~~~~~~~~~
2、use AndroidStudio Or Xcode  to open project and  compile project
In file included from E:/develop_client/axmol/extensions/scripting/lua-bindings/manual/fairygui/axlua_fairygui_conversions.cpp:1:
In file included from E:/develop_client/axmol/extensions/scripting/lua-bindings/manual/fairygui/axlua_fairygui_conversions.h:5:
In file included from E:/develop_client/axmol/extensions/fairygui/src\fairygui/FairyGUI.h:5:
In file included from E:/develop_client/axmol/extensions/fairygui/src\fairygui/UIPackage.h:5:
In file included from E:/develop_client/axmol/extensions/fairygui/src\fairygui/GObject.h:4:
In file included from E:/develop_client/axmol/extensions/fairygui/src\fairygui/Controller.h:6:
E:/develop_client/axmol/extensions/fairygui/src\fairygui/event/UIEventDispatcher.h:4:10: fatal error: 'FairyGUIMacros.h' file not found
#include "FairyGUIMacros.h"

--- a/extensions/scripting/lua-bindings/CMakeLists.txt
+++ b/extensions/scripting/lua-bindings/CMakeLists.txt
@@ -209,6 +209,7 @@ target_include_directories(${_AX_LUA_LIB}
     PRIVATE ${ax_root}/extensions/spine/src
     PRIVATE ${ax_root}/extensions/cocostudio/src
     PRIVATE ${ax_root}/extensions/fairygui/src
+    PRIVATE ${ax_root}/extensions/fairygui/src/fairygui
     PRIVATE ${ax_root}/extensions/Particle3D/src
     INTERFACE ${ax_root}/extensions/scripting/lua-bindings/manual
 )

--- a/extensions/scripting/lua-bindings/proj.android/CMakeLists.txt
+++ b/extensions/scripting/lua-bindings/proj.android/CMakeLists.txt
@@ -25,5 +25,6 @@ target_include_directories(${target_name}
   PRIVATE ../../../../thirdparty
   PRIVATE ../../../../thirdparty/robin-map/include
   PRIVATE ../../../../thirdparty/jni.hpp/include
+  PRIVATE ../../../../thirdparty/yasio
 )
 

